websites/W/Worldhideaway/presence.ts

```diff
@@ -151,7 +151,7 @@ presence.on("UpdateData", async () => {
 				if (check) {
 					presenceData.buttons = [
 						{
-							label: "Leyendo",
+							label: "Leer Un Articulo",
```

Comment:
This is more clear. In order to keep consistency with the other strings, could you remove the "Un"? ("un" would translated as "a", "Read a Article")